diff --git a/node_modules/strapi/lib/core/load-apis.js b/node_modules/strapi/lib/core/load-apis.js
index 0f7f985..470402b 100644
--- a/node_modules/strapi/lib/core/load-apis.js
+++ b/node_modules/strapi/lib/core/load-apis.js
@@ -15,7 +15,7 @@ module.exports = async ({ dir }) => {
     );
   }
 
-  const apis = await loadFiles(apiDir, '*/!(config)/**/*.*(js|json)');
+  const apis = await loadFiles(apiDir, '*/!(config)/**/*.*(js|json)', {globArgs: {ignore: ['**/__tests__/**/*.[jt]s?(x)', '**/tests/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)']}});
   const apiConfigs = await loadConfig(apiDir, '*/config/**/*.*(js|json)');
 
   return _.merge(apis, apiConfigs);
